app/forms/collection_form.rb

```diff
@@ -5,6 +5,14 @@
 class CollectionForm < DraftCollectionForm
   extend T::Sig
 
+  collection :contact_emails, populator: ContactEmailsPopulator.new(:contact_emails, ContactEmail),
+                              prepopulator: ->(*) { contact_emails << ContactEmail.new if contact_emails.blank? } do
+    property :id
+    property :email
+    property :_destroy, virtual: true
+    validates :email, format: { with: Devise.email_regexp }
```

Comment:
validate contact email on the collection form, note: the regex will NOT allow blanks